src/app/requirements-dev.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile requirements-dev.in
 #
--e git+https://github.com/beer-garden/brewtils@migrate_handler_optimization#egg=brewtils
+-e git+https://github.com/beer-garden/brewtils@develop#egg=brewtils
```

Comment:
If you merge in the latest instance of optimization_reverted this should already be done.